src/resources/views/ui/errors/4xx.blade.php

```diff
@@ -0,0 +1,16 @@
+@extends(backpack_view('errors.layout'))
+
+@php
+  $error_number ??= 400;
+@endphp
+
+@section('title')
+  {{ trans('backpack::base.error.'.$error_number) }}
+@endsection
+
+@section('description')
+  {!! $exception?->getMessage() ? e($exception->getMessage()) : trans('backpack::base.error.message_4xx', [
+    'href_back' => 'href="javascript:history.back()"',
+    'href_homepage' => 'href="'.url('').'"',
```

Comment:
Since this are the admin panel errors, wouldn't it made more sense to use `backpack_url()` here ?
```suggestion
    'href_homepage' => 'href="'.backpack_url().'"',
```

Reply:
me neither! 🤷 But let's keep it then.